From d2b79c60e945947cca19b3aff9443cf2411db3f4 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Sun, 18 Feb 2024 15:31:50 +0000
Subject: [PATCH 0901/1085] overlays: adau1977-adc: Replace use of i2c label

The label 'i2c' is no longer created by the firmware - i2c_arm or
i2c1 should be used instead. Replace the last occurrence of &i2c with
&i2c1.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/adau1977-adc-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/adau1977-adc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/adau1977-adc-overlay.dts
@@ -6,7 +6,7 @@
 	compatible = "brcm,bcm2835";
     
 	fragment@0 {
-        	target = <&i2c>;
+		target = <&i2c1>;
         	
 		__overlay__ {
 			#address-cells = <1>;
